selfdrive/car/docs_definitions.py

```diff
@@ -129,9 +129,10 @@ def get_column(self, column: Column, star_icon: str, footnote_tag: str) -> str:
     elif column == Column.MODEL and len(self.years):
       item += f" {self.years}"
 
-    footnote = get_footnote(self.footnotes, column)
-    if footnote is not None:
-      item += footnote_tag.format(self.all_footnotes[footnote])
+    footnotes = get_footnotes(self.footnotes, column)
+    if footnotes is not None:
+      for fn in footnotes:
+        item += footnote_tag.format(self.all_footnotes[fn])
```

Comment:
Can we split on ", " or ","?

Reply:
Sure

Reply:
I think we need to put the comma inside the superscript tag, somehow

Reply:
How do you like that?